pep_sphinx_extensions/pep_zero_generator/constants.py

```diff
@@ -55,5 +55,9 @@
 Many recent PEPs propose changes to Python's static type system
 or otherwise relate to type annotations.
 They are listed here for reference.
-"""
+""",
+    "governance": """\
+These PEPs detail Python's governance, including governance model proposals
+and selection, and the results of the annual steering council elections.
```

Comment:
```suggestion
These PEPs detail the Python community's governance,
including governance model proposals and selection,
and the results of the annual steering council elections.
```

I suggest modifying this to "the Python community's governance", since it includes more than just Python itself (e.g. the PyPA). Also, rewrapped accordingly

Reply:
Hmm, I think plain "Python's governance" is better, it covers things like Python the language, CPython the implementation, PyPA stuff.


"Python community" could be interpreted as covering things like Scientific Python, the giant Python discord, etc.

(But no strong preferences from me.)

Reply:
Right, this is where I turned the original "Python's language governance" text into just "Python's governance" to be less specific. That way it things like packaging governance PEPs don't feel wrong. But it also doesn't make a potentially overreaching claim.

Reply:
Hmm, well to me the current text still implies that is limited to Python the language, as opposed to independent organizations like the PyPA, but I can see the concerns, and how others could easily interpret it differently. Not a big deal, I suppose.